src/test/java/com/epam/tc/hw2/ex1/FirstExercise.java

```diff
@@ -0,0 +1,113 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BaseClass;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class FirstExercise extends BaseClass {
+
+    SoftAssertions softAssert = new SoftAssertions();
+
+    @Test
+    public void mainFirstTest() {
+        // 2. Assert Browser title
+        softAssert.assertThat(driver.getTitle()).isEqualTo("Home Page");
+        // 3. Perform login
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        WebElement caret = driver.findElement(By.xpath("//li[contains(@class, 'uui-profile-menu')]"));
+        caret.click();
+        WebElement loginInput = driver.findElement(By.xpath("//input[@id='name']"));
+        loginInput.sendKeys("Roman");
+        WebElement passwordInput = driver.findElement(By.xpath("//input[@id='password']"));
+        passwordInput.sendKeys("Jdi1234");
+        WebElement enterButton = driver.findElement(By.xpath("//button[contains(@class, 'btn-login')]"));
+        enterButton.click();
+        // 4. Assert Username is loggined
+        WebElement username = driver.findElement(By.id("user-name"));
+        softAssert.assertThat(username.getText()).isEqualTo("ROMAN IOVLEV");
+        // 5. Assert that there are 4 items on the header section are displayed, and they have proper texts
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
```

Comment:
There is no need to open the same page several times.
`driver.get("https://jdi-testing.github.io/jdi-light/index.html");` is used in setUp method and two times in this test.